config/templates/cloud-config-controller

```diff
@@ -693,6 +705,66 @@ write_files:
            $kubectl uncordon "$hostname"
            echo "done."'
 
+  {{if .Experimental.Plugins.Rbac.Enabled }}
+
+  #https://github.com/coreos/kube-aws/issues/230#issuecomment-274946933
+  - path: /srv/kubernetes/rbac/clusterrole.yaml
+    content: |
+        kind: ClusterRole
+        apiVersion: rbac.authorization.k8s.io/v1alpha1
+        metadata:
+            name: full-access
+        rules:
+          - apiGroups: ["*"]
+            resources: ["*"]
+            verbs: ["*"]
+          - nonResourceURLs: ["*"]
+            verbs: ["*"]
+
```

Comment:
Definitely no, if the same role already exist I'll ignore. 
Take a look on https://github.com/gianrubio/kube-aws/blob/394350c27440e60050fa3f01e21536c744e1bbf8/config/templates/cloud-config-controller#L384

Both versions will use the same role.